.compatibility_tests/compatibility_test_1_11/tests/integration/compatibility_1_11.rs

```diff
@@ -0,0 +1,200 @@
+// 1.3
+#[test_case(test_roqoqo_1_11::operations::ControlledRotateX::new(0,1, 1.0.into()).into(); "ControlledRotateX")]
+#[test_case(test_roqoqo_1_11::operations::ControlledRotateXY::new(0,1, 1.0.into(), 1.0.into()).into(); "ControlledRotateXY")]
+#[test_case(test_roqoqo_1_11::operations::ControlledControlledPauliZ::new(0,1,2).into(); "ControlledControlledPauliZ")]
+#[test_case(test_roqoqo_1_11::operations::ControlledControlledPhaseShift::new(0,1,2, 1.0.into()).into(); "ControlledControlledPhaseShift")]
+#[test_case(test_roqoqo_1_11::operations::Toffoli::new(0,1,2).into(); "Toffoli")]
+// 1.4
+#[test_case(test_roqoqo_1_11::operations::GPi::new(0, 0.1.into()).into(); "GPi")]
+#[test_case(test_roqoqo_1_11::operations::GPi2::new(0, 0.1.into()).into(); "GPi2")]
+// 1.5
+#[test_case(test_roqoqo_1_11::operations::PragmaControlledCircuit::new(0, test_roqoqo_1_11::Circuit::new()).into(); "PragmaControlledCircuit")]
+// Operations from 1.6
+#[test_case(test_roqoqo_1_11::operations::Squeezing::new(0, 0.1.into(), 0.1.into()).into(); "Squeezing")]
+#[test_case(test_roqoqo_1_11::operations::PhaseShift::new(0, 0.1.into()).into(); "PhaseShift")]
+#[test_case(test_roqoqo_1_11::operations::BeamSplitter::new(0, 1, 0.1.into(), 0.2.into()).into(); "BeamSplitter")]
+#[test_case(test_roqoqo_1_11::operations::PhotonDetection::new(0, "ro".into(), 0).into(); "PhotonDetection")]
+// Operations from 1.7
+#[test_case(test_roqoqo_1_11::operations::Identity::new(0).into(); "Identity")]
+// Operations from 1.8
+#[test_case(test_roqoqo_1_11::operations::PhaseDisplacement::new(0, 0.1.into(), 0.1.into()).into(); "PhaseDisplacement")]
+#[test_case(test_roqoqo_1_11::operations::EchoCrossResonance::new(0, 1).into(); "EchoCrossResonance")]
+#[test_case(test_roqoqo_1_11::operations::PragmaAnnotatedOp::new(test_roqoqo_1_11::operations::PauliX::new(0).into(), "test".to_string()).into(); "PragmaAnnotatedOp")]
+// Operations from 1.9 - nothing was added
+// Operations from 1.10
+// QuantumRabi, LongitudinalCoupling, JaynesCummings, SingleExcitationLoad, SingleExcitationStore and CZQubitResonator were all added
+// as unstable, but have been added as stable in 1.11
+// Operations from 1.11
+#[test_case(test_roqoqo_1_11::operations::QuantumRabi::new(0, 1, 0.1.into()).into(); "QuantumRabi")]
+#[test_case(test_roqoqo_1_11::operations::LongitudinalCoupling::new(0, 1, 0.1.into()).into(); "LongitudinalCoupling")]
+#[test_case(test_roqoqo_1_11::operations::JaynesCummings::new(0, 1, 0.1.into()).into(); "JaynesCummings")]
+#[test_case(test_roqoqo_1_11::operations::SingleExcitationLoad::new(0, 1).into(); "SingleExcitationLoad")]
+#[test_case(test_roqoqo_1_11::operations::SingleExcitationStore::new(0, 1).into(); "SingleExcitationStore")]
+#[test_case(test_roqoqo_1_11::operations::CZQubitResonator::new(0, 1).into(); "CZQubitResonator")]
+// Operations from 1.11 - uncomment for next unittests once made stable
```

Comment:
This line should be moved to under all the operations and the comment should be adapted